--- xf86-video-cirrus-1.5.1/src/alp_driver.c.0000~	2012-07-16 04:02:41.000000000 +0200
+++ xf86-video-cirrus-1.5.1/src/alp_driver.c	2013-04-06 02:28:28.165030786 +0200
@@ -39,9 +39,6 @@
 /* All drivers initialising the SW cursor need this */
 #include "mipointer.h"
 
-/* All drivers implementing backing store need this */
-#include "mibstore.h"
-
 #include "micmap.h"
 
 /* Needed by the Shadow Framebuffer */
@@ -1617,8 +1614,6 @@ AlpScreenInit(SCREEN_INIT_ARGS_DECL)
 	if (init_picture)
 		fbPictureInit (pScreen, 0, 0);
     
-	miInitializeBackingStore(pScreen);
-
 	/*
 	 * Set initial black & white colourmap indices.
 	 */
--- xf86-video-cirrus-1.5.1/src/lg_driver.c.0000~	2012-07-16 04:02:41.000000000 +0200
+++ xf86-video-cirrus-1.5.1/src/lg_driver.c	2013-04-06 02:28:35.078470154 +0200
@@ -46,9 +46,6 @@
 /* need this for inputInfo */
 #include "inputstr.h"
 
-/* All drivers implementing backing store need this */
-#include "mibstore.h"
-
 #include "micmap.h"
 
 /* Needed by the Shadow Framebuffer */
@@ -1373,8 +1370,6 @@ LgScreenInit(SCREEN_INIT_ARGS_DECL)
 
 	fbPictureInit(pScreen, 0, 0);
 
-	miInitializeBackingStore(pScreen);
-
 	/*
 	 * Set initial black & white colourmap indices.
 	 */
